theme/models.py

```diff
@@ -350,30 +353,17 @@ def get_quota_data(self):
         if qmsg is None:
             qmsg = QuotaMessage.objects.create()
 
-        enforce_quota = qmsg.enforce_quota
         soft_limit = qmsg.soft_limit_percent
-        hard_limit = qmsg.hard_limit_percent
-        today = datetime.date.today()
-        grace = self.grace_period_ends
         allocated = self.allocated_value
         unit = self.unit
-        used = self.get_used_value_by_zone(refresh=False)
```

Comment:
we aren't really using the `zone` anymore. Should we remove it?

Reply:
I thought about it, but if we wanted to stand up another minio instance, we could reuse zones for it. I setup the minio client alias to hydroshare, consistent with the only zone in irods that we had.